test/test_retryable_writes.py

```diff
@@ -454,6 +456,40 @@ def test_batch_splitting_retry_fails(self):
         self.assertEqual(coll.find_one(projection={'_id': True}), {'_id': 1})
 
 
+class TestWriteConcernError(IntegrationTest):
+    @client_context.require_version_min(3, 6)
+    @client_context.require_no_standalone
+    @client_context.require_no_mmap
+    @client_context.require_failCommand_fail_point
+    def test_RetryableWriteError_error_label(self):
+        listener = OvertCommandListener()
+        client = rs_or_single_client(
+            retryWrites=True, event_listeners=[listener])
+
+        fail_insert = {
```

Comment:
Can perform a successful insert before testing the failpoint behavior to ensure the collection is created first:
```
client.pymongo_test.testcoll.insert_one({})
```
AFAIK this shouldn't matter but its best practice to create the collection first.

Reply:
Done.